CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs

```diff
@@ -7,25 +7,29 @@ public class RemindLateDevelopers
     {
         private IGetLateDevelopers _getLateDevelopers;
         private IRemindDeveloper _remindDeveloper;
+        private IClock _clock;
 
         public RemindLateDevelopers(IGetLateDevelopers getLateDevelopers, IRemindDeveloper remindDeveloper, IClock clock)
         {
             _getLateDevelopers = getLateDevelopers;
             _remindDeveloper = remindDeveloper;
+            _clock = clock;
         }
 
         public void Execute(RemindLateDevelopersRequest remindLateDevelopersRequest)
         {
-            var lateDevelopers = _getLateDevelopers.Execute();
-            foreach (var lateDeveloper in lateDevelopers.Developers)
+            if ((_clock.Now().ToUnixTimeSeconds() % 1800 == 0) && (_clock.Now().Hour < 14))
```

Comment:
magic numbers

Reply:
Are you suggesting we remove the clever maths? 😢

Reply:
fixed in https://github.com/chubberlisk/crypto_tech/pull/17/commits/9d7c7cec01a3b7d3ac544e8b62e33b1226759a43